### DISCUSSION TITLE
Feedback a problem with TeamTalk Server

### DISCUSSION THREAD

**Discussion by yskAdmin** [BearWare/TeamTalk5#1757] [Q&A]
### Description

The file uploaded by non-administrator user shows that there is not enough disk space. I have authorized this user
Hope to solve this problem Thank you

### Application

- [X] qtTeamTalk
- [ ] TeamTalkAndroid
- [ ] iTeamTalk
- [ ] TeamTalkClassic
- [X] TeamTalkServer

### Platform

- [ ] Windows
- [ ] macOS
- [ ] Android
- [ ] iOS
- [X] Linux

### Actual behavior

The file uploaded by non-administrator user shows that there is not enough disk space. I have authorized this user
Hope to solve this problem Thank you

### Steps to reproduce problem

%100 recurrence

**Comment by bear101** [comment]
It sounds like the max disk usage and default channel quota have not been configured correctly when you ran the server wizard. You can see the two options in question here: https://github.com/BearWare/TeamTalk5/blob/master/Library/TeamTalkLib/test/tt5srv.xml_template#L24
